flink-connector/build.gradle.kts

```diff
@@ -66,6 +66,7 @@ dependencies {
   testImplementation(libs.junit.jupiter.api)
   testImplementation(libs.junit.jupiter.params)
   testImplementation(libs.mockito.core)
+  testImplementation(libs.mysql.driver)
```

Comment:
I'm curious about how the CI passed the previous tests if we don't have the driver. Has there been any change recently?

Reply:
Not changed, seems about the sequence of it, if some other its run first it may download MySQL drivers

Reply:
I see.